### DISCUSSION TITLE
关于 页眉双线和每章标题的间距

### DISCUSSION THREAD

**Discussion by BravoFr0st** [sjtug/SJTUThesis#936] [提问]
调节什么参数可以改变
- 页眉双线和 每章标题的间距？
- 左侧右侧页边距？

如图，我和我同学使用SJTUthesis（上）和教务处模板（下）的一些区别。
![2T@IUTR }TZLKOAE@5FP QY](https://user-images.githubusercontent.com/57180968/235347137-2761f455-70fc-4974-a6aa-1b1f1000ba3d.jpg)
![image](https://user-images.githubusercontent.com/57180968/235347253-b985c788-dd66-4ef5-b03d-090c2b2c03eb.png)

**Comment by BravoFr0st** [comment]
试着调整了这里的参数，但是似乎对第一个需求没有效果，可以达成第二个需求（`left,right`）。

```
\geometry
  {
    top           = 3.5 cm,
    bottom        = 4.0 cm,
    left          = 3 cm,%default=2.5
    right         = 3 cm,%default=2.5
    bindingoffset = 0.5 cm,
    headheight    = 1.5 cm,%default=1.5
    headsep       = 0.5 cm,
    footskip      = 1.0 cm
  }
```

**Reply by LogCreative** [reply]
改 `headsep` 参数，实际上这个间距的规定并不是强制的。

**Reply by zhl-1** [reply]
您好，尝试修改了headsep参数，但是好像并没有生效，即“页眉双线与下面正文的间距”

**Reply by LogCreative** [reply]
是可以生效的，或者直接在导言区结尾添加
```latex
\geometry{headsep=1.5cm}
```
